### ISSUE TITLE
[stable/ghost] <.name>: can't evaluate field name in type interface {}

### ISSUE THREAD

**Issue by gridworkz** [helm/charts#8820]
**Is this a request for help?**:
Yes,
---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG REPORT

**Version of Helm and Kubernetes**:
Tried on Helm 2.9.1 (and) 2.11.0 (and) 2.11-rc.3
Kubernetes 1.12.1

**Which chart**:
stable/ghost

**What happened**:
When running:
```
sudo helm install --name testapp \
    -f values.yaml \
    stable/ghost
```
The following error occurs:
```
Error: render error in "ghost/templates/ingress.yaml": template: ghost/templates/ingress.yaml:6:28: executing "ghost/templates/ingress.yaml" at <.name>: can't evaluate field name in type interface {}
```

**What you expected to happen**:
Successful chart installation

**How to reproduce it** (as minimally and precisely as possible):
Follow exact steps in **What happened**

**Anything else we need to know**:
Our values.yaml file:
```
ghostHost: ghost.test.cooby.online
ingress:
  enabled: true
  annotations:
    kubernetes.io/ingress.class: nginx
    kubernetes.io/tls-acme: "true"
  hosts:
    - ghost.test.cooby.online
  tls:
   - secretName: test-app-tls
     hosts:
       - ghost.test.cooby.online
mariadb:
  replication:
    enabled: true
```
Basically it's the last step in Alens' blog post here: [https://akomljen.com/get-automatic-https-with-lets-encrypt-and-kubernetes-ingress/](https://akomljen.com/get-automatic-https-with-lets-encrypt-and-kubernetes-ingress/)

Cheers, Dave

**Comment by gridworkz** [comment]
Alen updated the post to work with the new stable/ghost helm chart. All good now.
